index.html

```diff
@@ -1,43 +1,239 @@
-        </div>
-      </nav>
+        <nav>
+            <a class="logo">My logo</a>
+            <div>
+                <i class="fa-solid fa-bars menu-bar"></i>
+            </div>
+        </nav>
     </header>
 
     <section class="introduction">
-      <h1 class="title"> I'm Vishal, Glad to See You!</h1>
-      <p class="description">I'm a software developer! I can help you build a product, feature or website. Look
-        through some of my work and experience. If you like what you see and have a project you need coded, don't
-        hestiate to contact me.</p>
-  
-      <h2 class="social order">LET'S CONNECT</h2>
+        <h1 class="title"> I'm Vishal <br>Glad to See <br>You!</h1>
+        <p class="description">I'm a software developer! I can help you build a product, feature or website. Look
+            through some of my work and experience. If you like what you see and have a project you need coded, don't
+            hestiate to contact me.</p>
+
+        <h2 class="social order">LET'S CONNECT</h2>
+
+        <ul class="social-links">
+            <li><a href="https://twitter.com/Vismiss30" target="_blank" rel="noopener"><i
+                        class="fa-brands fa-twitter li1"></i></a></li>
+            <li><a href="https://www.linkedin.com/in/vishal-kumar-mishra-0a5226232/" target="_blank" rel="noopener"><i
+                        class="fa-brands fa-linkedin li2"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-m li3"></i></a></li>
+            <li><a href="https://github.com/Vishal30M" target="_blank" rel="noopener"><i
+                        class="fa-brands fa-github li4"></i></a></li>
+            <li><a href="#"><i class="fa-brands fa-angellist li5"></i></a></li>
+        </ul>
+
+    </section>
+
+    <section class="work-section">
+        <div class="section1">
+          <div class="right-block">
+            <img class="img-section1" src="image/snap3.png" alt="project image">
+          </div>
+          <div class="left-block">
+            <div class="experience">
+              <h1 class="section-title">Tonic</h1>
+              <ul class="list">
+                <li class="list1">CANOPY</li>
+                <li class="list2">Back End Dev</li>
+                <li class="list3">2015</li>
+              </ul>
+            </div>
+            <p class="section-description0">A daily selection of privately personalized reads; no accounts or
+              sign-ups
+              required.</p>
+            <ul class="tags">
+              <li class="li11">
+                <article class="text1">html</article>
+              </li>
+              <li class="li11">
+                <article class="text2">css</article>
+              </li>
+              <li class="li11">
+                <article class="text3">javascript
+              </li>
+            </ul>
+            <div class="action">
+              <button class="btn">See Project</button>
```

Comment:
- [ ] Just as requested by the previous reviewer, kindly adjust this section of your app. you can wrap them in a `div` element, then use flexbox to render them while giving them a uniform styling. 
# Figma:
![image](https://user-images.githubusercontent.com/50990925/212444182-37912d99-2b3d-490c-b418-5d89f1dc62c1.png)

# Your design :
![image](https://user-images.githubusercontent.com/50990925/212444197-5269fcd1-916d-4a3c-ab86-798f2fd7ce91.png)